.github/workflows/sonar.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - name: Calculate unit test coverage
         run: |
           pytest --cov-report xml:coverage-reports/coverage.xml --cov src tests/unit
+          sed -i "s/<source>\/home\/runner\/work\/firebolt-python-sdk\/firebolt-python-sdk<\/source>/<source>\/github\/workspace<\/source>/g" coverage-reports/coverage.xml
```

Comment:
You can use a different delimiter, for readability
`s#<source>/home/runner/work/firebolt-python-sdk/firebolt-python-sdk</source>#<source>/github/workspace</source>#g`